src/Nest/Aggregations/Aggregation.cs

```diff
@@ -1,32 +1,26 @@
+using Newtonsoft.Json;
 using System.Collections.Generic;
 
 namespace Nest
 {
-	/// <summary>
-	/// Describes an aggregation at request time when its being build
-	/// </summary>
-	[ExactContractJsonConverter(typeof(AggregationJsonConverter))]
-	public interface IAggregation
-	{
-	}
-
-	/// <summary>
-	/// Base for the OIS Aggregation DSL
-	/// </summary>
-	public interface IAggregationBase : IAggregation
```

Comment:
I'm not really sure of original reasoning, but we didn't have to.  I think the new separation is much cleaner.